Util/FileHandler.py

```diff
@@ -0,0 +1,39 @@
+import os
+import uuid
+
+from PIL import Image
+from PIL.ExifTags import GPSTAGS, TAGS
+
+
+def save(image: Image, filename: str) -> str:
+    # TODO replace with S3
+    if not os.path.exists('Data/'):
+        os.makedirs('Data')
+
+    _, ext = filename.split(".")
+    filename = uuid.uuid4().hex
+    image.save(f'Data/{filename}.{ext}')
+    return f'Data/{filename}.{ext}'
+
+
+def gps_details(image: Image) -> {}:
+    exif_table = {}
+    info = image.getexif()
+    for tag, value in info.items():
+        decoded = TAGS.get(tag, tag)
+        exif_table[decoded] = value
+
+    if 'GPSInfo' not in exif_table.keys():
```

Comment:
Anonymous plants get sent to [Null Island](https://en.wikipedia.org/wiki/Null_Island), nice.

Reply:
Perhaps it should fail if there's no attached coordinates, since in that case the image isn't useful to us?

Reply:
That's a shout, will update